fastmri/data/subsample.py

```diff
@@ -184,15 +217,41 @@ def __call__(
             pad = (num_cols - num_low_freqs + 1) // 2
             mask[pad : pad + num_low_freqs] = True
 
-            # determine acceleration rate by adjusting for the number of low frequencies
-            adjusted_accel = (acceleration * (num_low_freqs - num_cols)) / (
-                num_low_freqs * acceleration - num_cols
-            )
-            offset = self.rng.randint(0, round(adjusted_accel))
-
-            accel_samples = np.arange(offset, num_cols - 1, adjusted_accel)
-            accel_samples = np.around(accel_samples).astype(np.uint)
-            mask[accel_samples] = True
+            # If everything has been sampled in the center: we don't need to sample anything else.
+            if num_low_freqs * acceleration != num_cols:
```

Comment:
Do you want this instead?
```suggestion
            if num_low_freqs * acceleration >= num_cols:
```

Reply:
Aah yes, that would be even better.